testu01-stdin/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchurl, unzip, testu01 }:
+
+stdenv.mkDerivation rec {
+  name = "TestU01_stdin";
+  src = ./.;
+  nativeBuildInputs = [ testu01 ];
+  hardeningDisable = [ "format" ];
+  buildInputs = [];
+  configureFlags = [];
+  unpackPhase = "";
+  buildPhase = ''
+    gcc -std=c99 -O3 -fomit-frame-pointer -mtune=native TestU01_stdin.c -o TestU01_stdin -ltestu01 -lprobdist -lmylib -lm
+  '';
+  installPhase = ''
+    mkdir -p $out/bin
+    cp TestU01_stdin $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "Empirical Statistical Testing of Uniform Random Number Generators";
+    license = licenses.gpl3;
```

Comment:
@idontgetoutmuch, do you know what the license (and homepage) of this file are?

Reply:
They have disappeared. I found the code cached by google: http://webcache.googleusercontent.com/search?q=cache:Sw4Wm-UXUjsJ:www.freecx.co.uk/crypto/cryptanalysis/TestTools/TestU01/TestU01_stdin.c+&cd=2&hl=en&ct=clnk&gl=uk.

I guess we could write to the author?

Reply:
No luck - they have covered their tracks well